elliptic-curve/src/arithmetic.rs

```diff
@@ -26,7 +26,8 @@ pub trait CurveArithmetic: Curve {
         + PartialEq
         + Sized
         + Send
-        + Sync;
+        + Sync
+        + TryInto<NonIdentity<Self::AffinePoint>>;
```

Comment:
Perhaps notate the `Error` type on these?

```suggestion
        + TryInto<NonIdentity<Self::AffinePoint>, Error = Error>;
```

Reply:
Done.